src/practice3/task1_Serialization/Main.java

```diff
@@ -0,0 +1,72 @@
+package practice3.task1_Serialization;
+
+import practice1.task1.address.Address;
+import practice1.task1.address.Country;
+import practice1.task1.transport.FuelType;
+import practice1.task1.transport.Transport;
+import practice1.task1.transport.WheeledTransport;
+
+import java.io.*;
+
+public class Main {
+    public static void main(String[] args) {
+        try (ObjectOutputStream objectOutputStream = new ObjectOutputStream(new FileOutputStream("Address.dat"))) {
+            Address address = new Address(Country.BELARUS, "Vitebsk", "Komsomol'skaya", 17, 11);
+            objectOutputStream.writeObject(address);
+            System.out.println("Object Address is was write.");
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (NullPointerException e) {
+            e.printStackTrace();
+        } catch (SecurityException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
As you have same logic for processing all exceptions, you can combine them into one catch to reduce the amount of code

Reply:
Concatenated exceptions into one line.